src/status_im/ui/screens/popover/views.cljs

```diff
@@ -169,6 +170,9 @@
                    (= :transfer-multiaccount-unknown-error view)
                    [multiaccounts.key-storage/unknown-error-popover]
 
+                   (= :password-reset-success view)
```

Comment:
I didn't realise that. Definitely a better way.

Reply:
In this case, it's easier to use an id instead of the component itself. The success popover is a result of many effects and events. Passing the real view will be more work than this implementation.